### PR TITLE
Adds a Free Solarian Fleets ERT.

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Highly WIP, balancing needs to be done, playtesting to make sure it works, etc etc. I am highly liable to have fucked stuff up in the course of making this PR.

This PR also reworks the item descriptions for a few pieces of Sol gear, and changes the weighting for the EPMC ERT and Kataphract ERT.

dreamseeker_GuJQBRG8Iz

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!wip

**Comment by TheDanseMacabre** [comment]
!review

File: code/datums/ert/outsider.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 /datum/responseteam/kataphracts
 	name = "Kataphracts"
-	chance = 15
+	chance = 10
```
Comment (Geevies) [inline code/datums/ert/outsider.dm:8]: no bias here, but i don't think this is needed, i can review the rest of the PR if this is reverted

File: code/datums/outfits/ert/fsf_ert.dm [hunk 2]
```diff
@@ -0,0 +1,118 @@
+/datum/outfit/admin/ert/fsf_ert
+    name = "Free Solarian Fleets Marine"
+
+    uniform = /obj/item/clothing/under/rank/fatigues/marine
+    shoes = /obj/item/clothing/shoes/swat/ert
+    gloves = /obj/item/clothing/gloves/swat/ert
+    belt = /obj/item/storage/belt/military
+    back = /obj/item/storage/backpack/satchel_norm
+    head = /obj/item/clothing/head/helmet/space/void/sol
```
Comment (Geevies) [inline code/datums/outfits/ert/fsf_ert.dm:9]: if you're giving them a voidsuit, there's a proc i added to lancers that automatically sets up their voidsuits

Comment (Geevies) [reply code/datums/outfits/ert/fsf_ert.dm:9]: though it needs you to put an oxygen tank in their suit slot, so i don't think it's needed here

File: code/datums/outfits/ert/fsf_ert.dm [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      /obj/item/stack/medical/advanced/bruise_pack = 1,
+      /obj/item/stack/medical/advanced/ointment = 1,
+      /obj/item/reagent_containers/glass/bottle/thetamycin = 1
+    )
+
+/datum/outfit/admin/ert/fsf_ert/sapper
+    name = "Free Solarian Fleets Sapper"
+
+    back = /obj/item/storage/backpack/duffel
+    belt = /obj/item/storage/belt/utility/full
```
Comment (Geevies) [inline code/datums/outfits/ert/fsf_ert.dm:62]:
```suggestion
    belt = /obj/item/storage/belt/utility/very_full
```

Comment (Geevies) [reply code/datums/outfits/ert/fsf_ert.dm:62]: this one comes with all the gear you need

File: code/datums/outfits/ert/fsf_ert.dm [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    suit_store = /obj/item/gun/projectile/shotgun/pump/combat/sol
+    accessory = /obj/item/clothing/accessory/storage/brown_vest
+    accessory_contents = list(
+                  /obj/item/plastique = 5
+                )
+
+    backpack_contents = list(
+      /obj/item/melee/energy/sword/knife/sol = 1,
+      /obj/item/handcuffs/ziptie = 1,
+      /obj/item/device/multitool = 1,
```
Comment (Geevies) [inline code/datums/outfits/ert/fsf_ert.dm:74]: if you do the very_full belt, you can remove this

File: code/datums/outfits/ert/fsf_ert.dm [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+      /obj/item/weldingtool/hugetank = 1,
```
Comment (Geevies) [inline code/datums/outfits/ert/fsf_ert.dm:75]: this one too

File: code/datums/outfits/ert/fsf_ert.dm [hunk 6]
```diff
@@ -0,0 +1,124 @@
+/datum/outfit/admin/ert/fsf_ert
```
Comment (Arrow768) [inline code/datums/outfits/ert/fsf_ert.dm:1]: Remove the _ert suffix.
It's already clear that this is a ert from the path.

File: code/datums/outfits/ert/fsf_ert.dm [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+
+    uniform = /obj/item/clothing/under/rank/fatigues/marine
+    shoes = /obj/item/clothing/shoes/swat/ert
+    gloves = /obj/item/clothing/gloves/swat/ert
+    belt = /obj/item/storage/belt/military
+    back = /obj/item/storage/backpack/satchel_norm
+    head = /obj/item/clothing/head/helmet/space/void/sol
+    suit = /obj/item/clothing/suit/space/void/sol
+    suit_store = /obj/item/gun/projectile/automatic/rifle/sol
+    id = /obj/item/card/id/syndicate
```
Comment (Arrow768) [inline code/datums/outfits/ert/fsf_ert.dm:12]: The syndic card should only be used by "proper" antags as it allows self-modification.
This is not needed/wanted for a ert team.

File: code/datums/outfits/ert/fsf_ert.dm [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+        /obj/item/ammo_magazine/c762/sol = 1,
+        /obj/item/ammo_magazine/mc9mm = 2,
+        /obj/item/handcuffs/ziptie = 2,
+        /obj/item/grenade/frag = 1
+    )
+
+    accessory = /obj/item/clothing/accessory/holster/hip/brown
+    accessory_contents = list(/obj/item/gun/projectile/pistol/sol = 1)
+
+/datum/outfit/admin/ert/fsf_ert/get_id_access()
```
Comment (Arrow768) [inline code/datums/outfits/ert/fsf_ert.dm:31]: Same path issue as above.

File: code/datums/outfits/ert/fsf_ert.dm [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+	  return get_distress_access_lesser()
+
+/datum/outfit/admin/ert/fsf_ert/medic
```
Comment (Arrow768) [inline code/datums/outfits/ert/fsf_ert.dm:34]: Same path issue as above.

File: code/datums/outfits/ert/fsf_ert.dm [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+        /obj/item/reagent_containers/hypospray/cmo = 1,
+        /obj/item/reagent_containers/glass/bottle/inaprovaline = 1,
+        /obj/item/reagent_containers/glass/bottle/antitoxin = 1,
+        /obj/item/reagent_containers/glass/bottle/dexalin_plus = 1,
+        /obj/item/reagent_containers/glass/bottle/bicaridine = 1,
+        /obj/item/reagent_containers/glass/bottle/dermaline = 1,
+        /obj/item/reagent_containers/glass/bottle/perconol = 1
+    )
+
+/datum/outfit/admin/ert/fsf_ert/sapper
```
Comment (Arrow768) [inline code/datums/outfits/ert/fsf_ert.dm:63]: Same path issue as above.

File: code/datums/outfits/ert/fsf_ert.dm [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+      /obj/item/clothing/glasses/welding/superior = 1,
+      /obj/item/storage/box/shotgunshells = 1,
+      /obj/item/gun/projectile/pistol/sol = 1,
+      /obj/item/ammo_magazine/mc9mm = 2,
+      /obj/item/grenade/frag = 2
+    )
+
+    belt_contents = null
+
+/datum/outfit/admin/ert/fsf_ert/leader
```
Comment (Arrow768) [inline code/datums/outfits/ert/fsf_ert.dm:88]: Same path issue as above

File: code/datums/outfits/ert/fsf_ert.dm [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+    backpack_contents = list()
+
+    belt_contents = list(
+      /obj/item/ammo_magazine/c762/sol = 1,
+      /obj/item/melee/energy/sword/knife/sol = 1,
+      /obj/item/ammo_magazine/mc9mm = 2,
+      /obj/item/handcuffs/ziptie = 2
+    )
+	
+/datum/outfit/admin/ert/fsf_ert/synth
```
Comment (Arrow768) [inline code/datums/outfits/ert/fsf_ert.dm:106]: Same path issue as above

File: code/game/objects/items/weapons/cards_ids.dm [hunk 13]
```diff
@@ -498,6 +498,11 @@ var/const/NO_EMAG_ACT = -50
 	access = list(access_distress, access_maint_tunnels, access_external_airlocks)
 	..()
 
+/obj/item/card/id/distress/fsf_ert
```
Comment (Arrow768) [inline code/game/objects/items/weapons/cards_ids.dm:501]: If you set up a I'd card here then use it above aswell.

Also same path issue as above.

File: code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm [hunk 14]
```diff
@@ -0,0 +1,43 @@
+/datum/ghostspawner/human/ert/fsf_ert
```
Comment (Arrow768) [inline code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm:1]: Same path issue as above

File: code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+	name = "Free Solarian Fleets Marine"
+	short_name = "fsfr"
+	desc = "A responder from a Free Solarian Fleets marine fireteam."
+	welcome_message = "You're part of a marine fireteam dispatched from the FSFV Sforza, a Free Solarian Fleets destroyer on the SCC's payroll. Your primary motivation is money: the SCC pays you well, and you have a job to do."
+	max_count = 2
+	outfit = /datum/outfit/admin/ert/fsf_ert
+	mob_name_prefix = "Pfc. "
+	possible_species = list(SPECIES_HUMAN)
+
+/datum/ghostspawner/human/ert/fsf_ert/specialist
```
Comment (Arrow768) [inline code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm:11]: Same path issue as above

File: code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+	name = "Free Solarian Fleets Sapper"
+	short_name = "fsfeng"
+	desc = "An engineering specialist from a Free Solarian Fleets marine fireteam."
+	max_count = 1
+	outfit = /datum/outfit/admin/ert/fsf_ert/sapper
+	mob_name_prefix = "Cpl. "
+
+/datum/ghostspawner/human/ert/fsf_ert/specialist/med
```
Comment (Arrow768) [inline code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm:19]: Same path issue as above

File: code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+	name = "Free Solarian Fleets Medic"
+	short_name = "fsfmed"
+	desc = "A medical specialist from a Free Solarian Fleets marine fireteam."
+	outfit = /datum/outfit/admin/ert/fsf_ert/medic
+
+/datum/ghostspawner/human/ert/fsf_ert/leader
```
Comment (Arrow768) [inline code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm:25]: Same path issue as above

File: code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+	name = "Free Solarian Fleets Fireteam Leader"
+	short_name = "fsflead"
+	desc = "The leader of the Free Solarian Fleets marine fireteam."
+	max_count = 1
+	outfit = /datum/outfit/admin/ert/fsf_ert/leader
+	mob_name_prefix = "Sgt. "
+
+/datum/ghostspawner/human/ert/fsf_ert/synth
```
Comment (Arrow768) [inline code/modules/ghostroles/spawner/human/responseteams/fsf_ert.dm:33]: Same path issue as above

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Review by Arrow768 — CHANGES_REQUESTED** [review]
The CI-Tests fail.

**Review by Arrow768 — APPROVED** [review]

**Review by WickedCybs — APPROVED** [review]
Looks good and used this partly to learn how to my EPMC ERT revamp, cool.

**Review by Geevies — APPROVED** [review]

File: code/datums/outfits/ert/fsf.dm [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+
+    back = /obj/item/storage/backpack/duffel
+    belt = /obj/item/storage/belt/utility/very_full
+    gloves = /obj/item/clothing/gloves/yellow
+    back = /obj/item/storage/backpack/duffel/eng
+    suit_store = /obj/item/gun/projectile/shotgun/pump/combat/sol
+    accessory = /obj/item/clothing/accessory/storage/brown_vest
+    accessory_contents = list(
+                  /obj/item/plastique = 5
+                )
```
Comment (NonQueueingMatt) [inline code/datums/outfits/ert/fsf.dm:74]: This indentation is fucked.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
The FSF file is entirely space-indented. Use tabs.

**Pull request merged**